src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -1893,6 +1894,10 @@ public void testPublishUnpublishFeedbackSession() throws Exception {
         fsLogic.unpublishFeedbackSession(
                 sessionUnderTest.getFeedbackSessionName(), sessionUnderTest.getCourseId());
         
+        paramMap = createParamMapForAction(sessionUnderTest, EmailType.FEEDBACK_UNPUBLISHED);
+        EmailAction fsUnpublishedAction = new FeedbackSessionUnpublishedMailAction(paramMap);
+        fsUnpublishedAction.getPreparedEmailsAndPerformSuccessOperations();
```

Comment:
@tarun14110, I think you can add an assert here that `sentPublishedEmail` is false